masterlist.yaml

```diff
@@ -325,6 +325,14 @@ common:
       - lang: zh_CN
         text: '你已经安装了%1%，但是未发现F4SE！请查看F4SE的下载页面：[F4SE](http://f4se.silverlock.org)。'
 
+  - &requiresAAF
+    type: warn
+    content:
+      - lang: en
+        text: 'You have installed %1% which requires the Advanced Animation Framework, but it was not found! See the AAF download page: [AAF](https://www.nexusmods.com/fallout4/mods/31304).'
+      - lang: fr
+        text: "Vous avez installé %1% qui nécessite l'AAF, mais il n'a pas été trouvé! Voici la page de téléchargement d'AAF: [AAF](https://www.nexusmods.com/fallout4/mods/31304)."
+
```

Comment:
Aliases are usually added once they've become common.
But if you think it's going to be used a lot then it's fine to add it.

There is another option in an existing alias found in the SSE masterlist.

```yaml
  - &missingRequirementXforY
    type: warn
    content:
      - lang: en
        text: 'It appears you have installed **%2%**, but some of its requirements seem to be missing. Please ensure you have correctly installed **%1%**.'

  # ENBSeries
  - <<: *missingRequirementXforY
    subs:
      - '[ENBSeries](http://enbdev.com/download_mod_tesskyrimse.html)'
      - '[ENB Helper SE](https://www.nexusmods.com/skyrimspecialedition/mods/23174/)'
    condition: 'file("SKSE/Plugins/ENBHelperSE.dll") and not file("../d3d11.dll")'
```

Reply:
AAF is a very common dependency, I don't know why we'd do it any other way since it's just gonna be referenced a lot. I have fallout 4 uninstalled atm but I'll reinstall it and get to work on that eventually:tm:.